specifyweb/frontend/js_src/lib/components/QueryBuilder/Export.tsx

```diff
@@ -31,6 +36,8 @@ export function QueryExportButtons({
     | (() => RA<SerializedResource<SpQueryField>>)
     | undefined;
   readonly recordSetId: number | undefined;
+  readonly resultsArray: RA<QueryResultRow | undefined> | undefined;
```

Comment:
You don't need to include `Array` in variable names - instead, if variable name is plular, it's clear from that. I.e, you have `results` instead of `resutls` which signifies an array